Gaijinizer patch/Map085.txt

```diff
@@ -1262,60 +1339,67 @@ By receiving the engagement ring. \\C[2]I got engaged with Nina!#MTLed
 可愛いアイドルがマイルームに来ちゃったから
 動揺しちゃうのも分かる分かるぅ♡
 > CONTEXT: Map085/events/7/pages/0/12/Dialogue/enemy6
-Fufufu♪It is a good response! 
-Because a cute idol has come to my room
-I understand that I'll be upset#MTLed
+Fufufu. ♪ What a response!
+
+But I get it, you're all worked up because a
+cute idol's come to see you. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 言ったでしょ？
 
 『次に会うときがあったら、その時はよろしく』って！
 > CONTEXT: Map085/events/7/pages/0/17/Dialogue/enemy6
-Did you say that? "Best regards, hope we meet
-again"!#MTLed
+I told you, remember?
+
+That we'd meet again!#RonIffy
```

Comment:
We should check what we actually translated in 2-3 and put it here for consistencies's sake...

Reply:
The funny thing is, she never actually said that line or anything approaching that line that I can find.

Reply:
Can't find it either... Looks like we'll keep it as this.